github.sgnl.yaml

```diff
@@ -640,6 +654,579 @@ entities:
+        externalId: publishedAt
+        type: DateTime
+      - name: submittedAt
+        externalId: submittedAt
+        type: DateTime
+      - name: updatedAt
+        externalId: updatedAt
+        type: DateTime
+  # https://docs.github.com/en/graphql/reference/objects#pullrequestcommit
+  PullRequestCommit:
+    displayName: PullRequestCommit
+    externalId: PullRequestCommit
+    description: Entity representing a Commit in a GitHub PullRequest
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: pullRequestId
+        externalId: $.pullRequest.id
+        type: String
+        indexed: true
+        attributeAlias: commitPullRequestId
+      - name: commitId
+        externalId: $.commit.id
+        type: String
+      - name: committedDate
+        externalId: committedDate
+        type: DateTime
+      - name: commitAuthor
+        externalId: $.commit.author.user.id
+        type: String
+        indexed: true
+        attributeAlias: commitAuthorId    
+  # https://docs.github.com/en/graphql/reference/objects#pullrequestchangedfile
+  PullRequestChangedFile:
+    displayName: PullRequestChangedFile
+    externalId: PullRequestChangedFile
+    description: Entity representing a changed file in a GitHub PullRequest
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: uniqueId # concatenated the file path and PR id
+        externalId: uniqueId
+        type: String
+        uniqueId: true
+      - name: pullRequestId
+        externalId: pullRequestId
+        type: String
+        indexed: true
+        attributeAlias: changedFilePullRequestId
+      - name: path # unique for each PR
+        externalId: path
+        type: String
+        indexed: true
+      - name: additions
+        externalId: additions
+        type: Int64
+      - name: deletions
+        externalId: deletions
+        type: Int64
+      - name: changeType
+        externalId: changeType
+        type: String
+      - name: viewerViewedState
+        externalId: viewerViewedState
+        type: String
+  
+  # https://docs.github.com/en/graphql/reference/objects#issue
+  Issue:
+    displayName: Issue
+    externalId: Issue
+    description: Entity representing a GitHub Issue
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+        attributeAlias: issueId
+      - name: authorLogin
+        externalId: $.author.login
+        type: String
+        indexed: true
+        attributeAlias: issueAuthorLogin
+      - name: repositoryId
+        externalId: $.repository.id
+        type: String
+        indexed: true
+        attributeAlias: issueRepositoryId
+      - name: activeLockReason
+        externalId: activeLockReason
+        type: String
+      - name: closed
```

Comment:
To index?
"Deny access if assigned issue is closed"

Reply:
added